pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -38,7 +38,7 @@ stdenv.mkDerivation rec {
       "http://teamspeak.gameserver.gamed.de/ts3/releases/${version}/TeamSpeak3-Client-linux_${arch}-${version}.run"
       "http://files.teamspeak-services.com/releases/${version}/TeamSpeak3-Client-linux_${arch}-${version}.run"
     ];
-    sha256 = if stdenv.is64bit 
+    sha256 = if stdenv.is64bit
```

Comment:
Your local build probably used the cached fetchurl derivation based on the sha256